generators/search-map.js

```diff
@@ -4,7 +4,8 @@ var fs = require('fs'),
 	Q = require('q'),
 	writeFile = Q.denodeify(fs.writeFile),
 	mkdirs = Q.denodeify(require("fs-extra").mkdirs),
-	sanitizeHtml = require('sanitize-html');
+	markdown = require('github-flavored-markdown'),
```

Comment:
Actually, looks like they refactored it and moved it to its own repo: https://github.com/jgm/commonmark.js